MonstieWash/Assets/Scripts/UIManager.cs

```diff
@@ -36,19 +50,105 @@ private void ToggleUIVisibility()
         CBAnimator.SetBool("Hide", m_UIVisible);
     }
 
-    // Update is called once per frame
-    private void Update()
+/// <summary>
+/// Public method used by TaskTracker to initialise clipboard after all task trackers have been initialised. Creates filters for tasks to show on each scene.
+/// </summary>
+/// <param name="keys">A list of all the keys used to identify tasks tracked in TaskTracker</param>
+    public void LoadKeys(List<string> keys)
+    {
+        m_TaskKeys = keys;
+        foreach (var scene in m_roomSaver.m_allScenes)
```

Comment:
Also since we're changing `m_allScenes` to a readonly property this will need to be adjusted.
```suggestion
        foreach (var scene in m_roomSaver.AllScenes)
```